### DISCUSSION TITLE
[Script request] Bluesky Social

### DISCUSSION THREAD

**Discussion by jsapede** [community-scripts/ProxmoxVE#276] [Request script]
### Application Name

Bluesky socila

### Website

https://github.com/bluesky-social

### Description

Selfhosted fedrated social media alternative to twitter/x

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by mzb2xeo** [comment]
They have a pretty good guide and scripts to install it on this site: 

https://github.com/bluesky-social/pds
